app/views/users/shared/otp_visible.js.erb

```diff
@@ -0,0 +1,8 @@
+$("#otp").css({ visibility: 'visible', display: 'block' }) 
```

Comment:
@ajaystha42 as I mentioned [here](https://github.com/restarone/violet_rails/pull/1153/files#r996319013) -- this file is duplicated. Please DRY it up

Reply:
sure @donrestarone